serving/samples/guestbook-redis-go/README.md

```diff
@@ -0,0 +1,154 @@
+# Deploying Redis alongside a Knative Service
+
+A simple web app written in Go that demonstrates how to deploy Redis (using
+vanilla Kubernetes components) alongside with a Knative Service that relies on
+it. The guestbook application shows a page with a form that allows users to
+leave a message under a name of their choosing. Names and messages map to keys
+and values in Redis, so only one message per user is saved at a time.
+
+## Prerequisites
+
+1. [Install Knative Serving](https://github.com/knative/docs/blob/master/install/README.md)
```

Comment:
Use relative links when possible.

Reply:
Done. Also fixed other links where this applies.